services/venice-router/src/main/java/com/linkedin/venice/router/api/DictionaryRetrievalService.java

```diff
@@ -137,6 +139,27 @@ public void handleStoreChanged(Store store) {
     }
   };
 
+  enum LogLevel {
```

Comment:
Shall we use `Level` from log4j instead?

Reply:
Modified to use log4j.